docs/source/specification.md

```diff
@@ -4,7 +4,9 @@
 
 <!-- What this document intends to do (and what it doesn't), the level of detail we aim for contrasted with other technical standards -->
 
-The SATRE specification follows a capability-evaluation model.
+The SATRE specification is our attempt to crowd-source, compile, and document knowledge, best practices and capabilities around TRE building and operation from different institutions; this includes all aspects of TRE provision such as information governance procedures, computing technology, data management and other capabilities.
+
+The specification follows a capability-evaluation model.
```

Comment:
Do you have a one-line explanation of what "capability-evaluation model" means?

Reply:
I think the explanation is the following sentence - as such I'm tempted to remove this line entirely